### ISSUE TITLE
The requested module does not provide an export named 'default'

### ISSUE THREAD

**Issue by hitecSmartHome** [GuptaSiddhant/react-broadcast-channel#30]
Hi! I have got this error when I run the app.

```js
Uncaught SyntaxError: The requested module '/node_modules/.vite/deps/use-broadcast-channel.js?v=3a2c67f1' does not provide an export named 'default' (at NotificationRequest.tsx:15:1)
```

How I try to use it

```js
import { useBroadcastChannel } from "use-broadcast-channel";

function SubChange() {
  useBroadcastChannel("testBroadcast", async function (e: any) {
    try {
      const resp = await api.post("/test", {
        subscription: JSON.stringify(e.detail.subData),
      });
      console.log(resp);
    } catch (e) {
      console.error(e);
    }
  });
  return null;
}
```

I have installed the latest with npm

Also in the `package.json` file I got a `0.1.3` version and not a `1.0.3`

```
"use-broadcast-channel": "^0.1.3",
```

**Comment by GuptaSiddhant** [comment]
Hi @hitecSmartHome , could you create a small reproduction on CodeSandbox or another platform?

**Comment by GuptaSiddhant** [comment]
Fixed in v0.3.0
